Doc/library/sqlite3.rst

```diff
@@ -1205,60 +1216,51 @@ you can let the :mod:`sqlite3` module convert SQLite types to different Python
 types via converters.
 
 
-Using adapters to store additional Python types in SQLite databases
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Using adapters to store custom Python types in SQLite databases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-As described before, SQLite supports only a limited set of types natively. To
-use other Python types with SQLite, you must **adapt** them to one of the
-sqlite3 module's supported types for SQLite: one of NoneType, int, float,
-str, bytes.
+SQLite supports only a limited set of types natively.
+To store custom Python types in SQLite databases, *adapt* them to one of the
```

Comment:
I'm not sure that changing 'other' and 'additional' to 'custom' makes it clearer. To me, 'custom Python types' sounds like it is referring specifically to user-defined classes.

Reply:
Good observation. I'll revert those wordings.

Reply:
Actually, reading through the changes once more, it seems that 'custom' is used for adapters and converters everywhere else but these few lines. So, changing these to 'custom' actually normalises the wording, and makes these lines consistent with the rest of the docs.

So, I'm +1 for keeping the change as it is.